pages/models/generations/Sources/VSC/EMTGridFollowingVSC/index.md

```diff
@@ -0,0 +1,510 @@
+     <td style="border-left:2px solid black;">$$1$$</td>
+     <td>$$0$$</td>
+     <td>$$0$$</td>
+     <td style="border-left:2px solid black;">$$\frac{2E_{DC}}{3}$$</td>
+     <td>$$-\frac{E_{DC}}{3}$$</td>
+     <td style="border-right:2px solid black;">$$-\frac{E_{DC}}{3}$$</td>
+</tr>
+<tr>
+     <td style="border-left:2px solid black;">$$2$$</td>
+     <td style="border-left:2px solid black;">$$1$$</td>
+     <td>$$1$$</td>
+     <td>$$0$$</td>
+     <td style="border-left:2px solid black;">$$\frac{E_{DC}}{3}$$</td>
+     <td>$$\frac{E_{DC}}{3}$$</td>
+     <td style="border-right:2px solid black;">$$-\frac{2E_{DC}}{3}$$</td>
+</tr>
+<tr>
+     <td style="border-left:2px solid black;">$$3$$</td>
+     <td style="border-left:2px solid black;">$$0$$</td>
+     <td>$$1$$</td>
+     <td>$$0$$</td>
+     <td style="border-left:2px solid black;">$$-\frac{E_{DC}}{3}$$</td>
+     <td>$$\frac{2E_{DC}}{3}$$</td>
+     <td style="border-right:2px solid black;">$$-\frac{E_{DC}}{3}$$</td>
+</tr>
+<tr>
+     <td style="border-left:2px solid black;">$$4$$</td>
+     <td style="border-left:2px solid black;">$$0$$</td>
+     <td>$$1$$</td>
+     <td>$$1$$</td>
+     <td style="border-left:2px solid black;">$$-\frac{2E_{DC}}{3}$$</td>
+     <td>$$\frac{E_{DC}}{3}$$</td>
+     <td style="border-right:2px solid black;">$$\frac{E_{DC}}{3}$$</td>
+</tr>
+<tr>
+     <td style="border-left:2px solid black;">$$5$$</td>
+     <td style="border-left:2px solid black;">$$0$$</td>
+     <td>$$0$$</td>
+     <td>$$1$$</td>
+     <td style="border-left:2px solid black;">$$-\frac{E_{DC}}{3}$$</td>
+     <td>$$-\frac{E_{DC}}{3}$$</td>
+     <td style="border-right:2px solid black;">$$\frac{2E_{DC}}{3}$$</td>
+</tr>
+<tr>
+     <td style="border-left:2px solid black;">$$6$$</td>
+     <td style="border-left:2px solid black;">$$1$$</td>
+     <td>$$0$$</td>
+     <td>$$1$$</td>
+     <td style="border-left:2px solid black;">$$\frac{E_{DC}}{3}$$</td>
+     <td>$$-\frac{2E_{DC}}{3}$$</td>
+     <td style="border-right:2px solid black;">$$\frac{E_{DC}}{3}$$</td>
+</tr>
+<tr style="border-bottom:2px solid black;">
+     <td style="border-left:2px solid black;">$$7$$</td>
+     <td style="border-left:2px solid black;">$$1$$</td>
+     <td>$$1$$</td>
+     <td>$$1$$</td>
+     <td style="border-left:2px solid black;">$$0$$</td>
+     <td>$$0$$</td>
+     <td style="border-right:2px solid black;">$$0$$</td>
+</tr>
+</table>
+
+For instance, if the voltage setpoint is located in the region delimited by states $$0/7-1-6$$, the switching order would be $$000 \rightarrow 100 \rightarrow 101 \rightarrow 111 \rightarrow 101 \rightarrow 100 \rightarrow 000$$, or, if we use state identifiers, $$0 \rightarrow 1 \rightarrow 6 \rightarrow 7 \rightarrow 6 \rightarrow 1 \rightarrow 0$$. This means that the duty cycles are split in two for all the states except the 7-state. The following figure represents a switching state of the mentioned pattern:
```

Comment:
it there a function/algorithm that we can write explicitely to describe the switching sequence?